src/widgets/Container/index.js

```diff
@@ -3,17 +3,34 @@ import Markdown from '../Markdown';
 import styles from './styles.css';
 
 export default class Container extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      jsonView: false
+    }
+
+    this.handleOnClick = this.handleOnClick.bind(this);
+  }
+
+  handleOnClick(event) {
```

Comment:
You can remove the need of `this.handleOnClick = this.handleOnClick.bind(this);` by rewriting it as:

```js
handleOnClick = (event) => {

};
```